tower-http/src/cors/allow_private_network.rs

```diff
@@ -84,6 +84,15 @@ impl From<bool> for AllowPrivateNetwork {
     }
 }
 
+impl<F> From<F> for AllowPrivateNetwork
+where
+    F: Fn(&HeaderValue, &RequestParts) -> bool + Send + Sync + 'static,
+{
+    fn from(f: F) -> Self {
+        AllowPrivateNetwork::predicate(f)
```

Comment:
The other `Allow` types that have a `predicate` constructor don't have such `From` impls. I don't think we should add one for `AllowPrivateNetwork`. I'm pretty sure it will degrade the error message when a non-matching type is passed to a function expecting `impl Into<AllowPrivateNetwork>`.

Reply:
Yes, I see that with this `From` impl the error message gets worse and for example no longer recommends that _`From<bool>` is implemented for `AllowPrivateNetwork`_. Removed it.